typespec/radius/v1/trackedresource.tsp

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+    
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import "@typespec/openapi";
+
+using OpenAPI;
+
+/* Using Azure.ResoureManager namespace here is a workaround for typespec bug: https://github.com/Azure/typespec-azure/issues/110" */
+namespace Azure.ResourceManager;
+ 
```

Comment:
Can you please add the comment why we need this ? Once you add it, I will approve :) Thanks for this work.